aspnetcore/toc.yml

```diff
@@ -688,8 +688,6 @@ items:
         uid: web-api/advanced/conventions
       - name: Handle errors
         uid: web-api/handle-errors
-      - name: Port tunneling in VS to debug web APIs
-        href: /connectors/custom-connectors/port-tunneling
```

Comment:
This is the API with controllers section. Should I revert this?
<img width="191" alt="image" src="https://user-images.githubusercontent.com/3605364/200496881-dd1e30ea-1c75-4c87-b01f-c835459f7827.png">

Reply:
Do we need a TOC on API testing and including both HttpRepl, Port tunneling?